Framework/include/QualityControl/AggregatorRunnerFactory.h

```diff
@@ -36,7 +37,7 @@ class AggregatorRunnerFactory
   AggregatorRunnerFactory() = default;
   virtual ~AggregatorRunnerFactory() = default;
 
-  static framework::DataProcessorSpec create(AggregatorRunnerConfig arc, std::vector<AggregatorConfig> acs);
+  static framework::DataProcessorSpec create(const o2::quality_control::core::InfrastructureSpec& infrastructureSpec);
```

Comment:
I don't like that it requres the whole `InfrastructureSpec` to go forward. How about reducing it to what it actually needs?
Also, requiring `InfrastructureSpec` instead of `Configs` makes an exception with regards to other QC devices, but perhaps it cannot be done in a different way.

Reply:
Yes, I agree, I did not realize that. I changed it.